lib/room.rb

```diff
@@ -0,0 +1,34 @@
+STANDARD_RATE = 200
+
+module BookingSystem
+  class Room
```

Comment:
Big design thought: What do you get from a `Room` that you couldn't get from a Hash of Arrays?

This implementation is totally valid, but it's an interesting thought exercise!